src/features/surveys/l10n/messageIds.ts

```diff
@@ -30,6 +30,20 @@ export default makeMessages('feat.surveys', {
         select: m('Single-choice (drop-down)'),
       },
     },
+    deleteDialog: {
```

Comment:
The `deleteDialog` is not a "block". The blocks are the various types of questions and text block. I think you should move `deleteDialog` up one level.

Also split it up, because the `deleteBlock` and `deleteQuestionOpt` are actually two separate dialogs IMO.

In the end, you should have `messageIds.deleteBlockDialog` and `messageIds.deleteOptionDialog` or something like that.

Reply:
thought I added fixed messageId, but apparently not. Let me fix it!